pos_cashbox/static/src/js/tour.js

```diff
@@ -0,0 +1,25 @@
+odoo.define('pos_cashbox.tour', function(require) {
+    "use strict";
+
+    var tour = require('web_tour.tour');
+
+    tour.register('pos_cashbox_tour', {
+        test: true,
+        url: "/web",
+    }, [{
+        trigger: '.o_app[data-menu-xmlid="point_of_sale.menu_point_root"], .oe_menu_toggler[data-menu-xmlid="point_of_sale.menu_point_root"]',
+        content: "Ready to launch your <b>point of sale</b>? <i>Click here</i>.",
+        position: 'bottom',
+    }, {
+        trigger: ".o_pos_kanban button.oe_kanban_action_button",
+        content: "<p>Click to start the point of sale interface. It <b>runs on tablets</b>, laptops, or industrial hardware.</p><p>Once the session launched, the system continues to run without an internet connection.</p>",
+        position: "bottom"
+    }, {
+        trigger: '.o_main_content:has(.loader:hidden)',
+        content: 'waiting for loading to finish',
+        run: function () {
+            // it's a check
```

Comment:
Use more clear comment, e.g.

    // specify empty run method to don't click on trigger